pkg/tuned/controller.go

```diff
@@ -1486,6 +1638,10 @@ func RunInCluster(stopCh <-chan struct{}, version string) error {
 		panic(err.Error())
 	}
 
+	if err := os.MkdirAll(ocpTunedRunDir, os.ModePerm); err != nil {
+		panic(err.Error())
```

Comment:
Is there a reason why not returning the error?

Reply:
the biggest/only reason is following what the surrounding code is doing